### PR TITLE
Add container support both for IoT Edge Dev Tool and IoT Edge Runtime

### PR DESCRIPTION
**Pull request by yorek**
Improved docker folder structure to keep the runtime and the tool in two separate folder
Added support for running IoT Edge Agent in a container

### CODE REVIEW

File: docker/runtime/readme.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+
+### Run docker image
+
+Create and set the enviroment variable IOT_DEVICE_CONNSTR to your IoT Edge Device connection string.
+
+Execute the `run-container.ps1` file. A container named `iotedgec` will be created and inside the `iotedged` daemon will be started. If everything is working correctly you will see the deamon output log.
+
+The deamon will use the default ip address assigned by Docker to the container (most probably will be 172.17.0.2). The discovery is automatically done in the `rund.sh` script. If you have changed your Docker network configuration this script may not work as expected, as it looks specifically for `eth0`.
+
+### Use IoT Edge tool
```
Comment (jongio) [inline docker/runtime/readme.md]: Shouldn't this be use IoT Edge CLI?

File: docker/runtime/run-container.ps1 [hunk 2]
```diff
@@ -0,0 +1,18 @@
+if (-not (Test-Path env:IOT_DEVICE_CONNSTR)) { 
+    Write-Host "Cannot run IoT Edge container: IOT_DEVICE_CONNSTR is not set"
+    Write-Host "Eg:"
+    Write-Host "`$env:IOT_DEVICE_CONNSTR=`"HostName=iothub0730.azure-devices.net;DeviceId=myEdgeDevice;SharedAccessKey=zfD73oX3agHTlT0rOvjPnYTkxRPw/k3U0exEGBDWQ5A=`""
```
Comment (jongio) [inline docker/runtime/run-container.ps1]: This is not an actual secret right?

File: docker/runtime/readme.md [hunk 3]
```diff
@@ -0,0 +1,48 @@
+# Running Azure IoT Edge in a container
+
+## Notes
+
+The following procedure has been tested on Windows 10 machine only. To avoid unexpected application crashes, run everything from Powershell prompts.
```
Comment (jongio) [inline docker/runtime/readme.md]: Didn't someone add support for Mac / Linux?

File: docker/runtime/run-container.ps1 (hunk 2 above)
Comment (yorek) [reply docker/runtime/run-container.ps1]: Right, device has been removed

File: docker/runtime/readme.md (hunk 1 above)
Comment (yorek) [reply docker/runtime/readme.md]: Corrected

File: docker/runtime/readme.md (hunk 3 above)
Comment (yorek) [reply docker/runtime/readme.md]: Yep, removed the "Note" section, it's not needed anymore

**Pull request merged**